File: 1704-determine-if-string-halves-are-alike/1704-determine-if-string-halves-are-alike.cpp
class Solution {
public:
    bool halvesAreAlike(string s) {
        string ss="aeiouAEIOU";
        int count=0;
        for(int i=0;i<s.size();i++)
        {
            bool ans=false;
            for(char c:ss)
            {
                if(c==s[i])
                {
                    ans=true;
                    break;
                }
            }
            if(ans)
            {
                if(i<s.size()/2)count++;
                else count--;
            }
        }
        return count==0;
    }
};

File: 1704-determine-if-string-halves-are-alike/1704-determine-if-string-halves-are-alike.py
class Solution:
    def halvesAreAlike(self, s: str) -> bool:
        k=set(['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U'])
        a=0
        b=0
        for i in range(len(s)//2):
            if s[i]in k:
                a+=1
        for i in range(len(s)//2,len(s)):
            if s[i]in k:
                b+=1
        return a==b